tests/integration/cli/tests/run.rs

```diff
@@ -263,104 +274,117 @@ fn run_wasi_works_non_existent() -> anyhow::Result<()> {
+        .stderr(contains("wapm_source: Querying the GraphQL API"))
+        .stderr(contains("builtin_loader: Downloading a webc file"))
+        .stderr(contains("module_cache::filesystem: Saved to disk"));
 
     let assert = Command::new(get_wasmer_path())
         .arg("python/python")
-        .arg(format!("--mapdir=.:{}", asset_path().display()))
+        .arg(format!("--mapdir=/app:{}", asset_path().display()))
         .arg("--registry=wasmer.io")
-        .arg("test.py")
-        .env("WASMER_DIR", wasmer_dir.path())
-        .env("RUST_LOG", &rust_log)
-        .assert();
+        .arg("/app/test.py")
+        .env("WASMER_CACHE_DIR", wasmer_dir.path())
+        .env("RUST_LOG", &*CACHE_RUST_LOG)
+        .assert()
+        .success();
 
-    assert.success().stderr("asdf");
+    assert
+        .stderr(contains("wapm_source: Cache hit!"))
+        .stderr(contains("builtin_loader: Cache hit!"))
+        .stderr(contains("module_cache::filesystem: Cache hit!"));
 }
 
 #[test]
 fn run_test_caching_works_for_packages_with_versions() {
+    let wasmer_dir = TempDir::new().unwrap();
+
     let assert = Command::new(get_wasmer_path())
         .arg("python/python@0.1.0")
         .arg(format!("--mapdir=/app:{}", asset_path().display()))
         .arg("--registry=wasmer.io")
         .arg("/app/test.py")
+        .env("RUST_LOG", &*CACHE_RUST_LOG)
+        .env("WASMER_CACHE_DIR", wasmer_dir.path())
         .assert()
         .success();
 
-    assert.stdout("hello\n");
+    assert
+        .success()
+        .stderr(contains("wapm_source: Querying the GraphQL API"))
+        .stderr(contains("builtin_loader: Downloading a webc file"))
+        .stderr(contains("module_cache::filesystem: Saved to disk"));
 
     let assert = Command::new(get_wasmer_path())
         .arg("python/python@0.1.0")
         .arg(format!("--mapdir=/app:{}", asset_path().display()))
         .arg("--registry=wasmer.io")
         .arg("/app/test.py")
-        .env(
-            "RUST_LOG",
-            "wasmer_wasix::runtime::package_loader::builtin_loader=debug",
-        )
+        .env("RUST_LOG", &*CACHE_RUST_LOG)
+        .env("WASMER_CACHE_DIR", wasmer_dir.path())
         .assert();
 
     assert
         .success()
-        // it should have ran like normal
-        .stdout("hello\n")
-        // we hit the cache while fetching the package
-        .stderr(contains(
-            "builtin_loader: Cache hit! pkg.name=\"python\" pkg.version=0.1.0",
-        ));
+        .stderr(contains("wapm_source: Cache hit!"))
+        .stderr(contains("builtin_loader: Cache hit!"))
+        .stderr(contains("module_cache::filesystem: Cache hit!"));
 }
 
-// FIXME: Re-enable. See https://github.com/wasmerio/wasmer/issues/3717
-#[ignore]
 #[test]
 fn run_test_caching_works_for_urls() {
+    let wasmer_dir = TempDir::new().unwrap();
+
     let assert = Command::new(get_wasmer_path())
+        .arg("run")
         .arg("https://wapm.io/python/python")
```

Comment:
still targetting "wapm.io"?

Reply:
Ah good point, I'll go through and update everything to use either `wasmer.io` or `wasmer.wtf`.